person.rb

```diff
@@ -0,0 +1,22 @@
+class Person
+  attr_reader :id
+  attr_accessor :name, :age
+
+  # constructor
+  def initialize(id, age, parent_permision, name = 'unknown')
+    @id = id
```

Comment:
- [ ] As per this [requirement](https://github.com/microverseinc/curriculum-ruby/blob/main/oop/school_library_people.md#:~:text=parent_permission%20as%20parameter.-,name%20and%20parent_permission%20are%20optional%20and%20have%20default%20values%20of%20%22Unknown%22%20and%20true.,-Getters%20for%20%40id), The `parent_permission` parameter has a default value `true`. Please add it.

- [ ] [Optional] [Highly recommended] It's a good practice and professional to generate the `id` automatically like the suggestion below, not as a parameter as you did here.
```ruby
def initialize(age, name = "Unknown", parent_permission = true)
    @id = Random.rand(1..1000)
    @age = age
    @name = name
    @parent_permission = parent_permission
  end
```

Reply:
Hi @Fombi-Favour,
try `parent_permission: true` instead of `parent_permission = true`